momento-sdk/src/main/java/momento/sdk/messages/CacheDictionaryFetchResponse.java

```diff
@@ -15,20 +15,20 @@ class Hit implements CacheDictionaryFetchResponse {
     private final Map<ByteString, ByteString> fieldsToValues;
 
     /**
-     * Constructs a dictionary fetch hit with a list of encoded keys and values.
+     * Constructs a dictionary fetch hit with a map of encoded fields to values.
      *
-     * @param fieldsToValues the retrieved dictionary.
+     * @param fieldsToValues The retrieved map.
      */
     public Hit(Map<ByteString, ByteString> fieldsToValues) {
       this.fieldsToValues = fieldsToValues;
     }
 
     /**
-     * Gets the retrieved values as a dictionary of UTF-8 string keys and values.
+     * Gets the retrieved elements as a map of UTF-8 string fields to UTF-8 string values.
      *
-     * @return the dictionary.
+     * @return The map.
      */
-    public Map<String, String> valueDictionaryStringString() {
+    public Map<String, String> elementsStringString() {
```

Comment:
Do we not prefix the payload we `value`?

Reply:
I'm not sure I understand. I changed this to elements because the map is a combination of fields and values. I think that 'dictionary' was in there because .NET has a Dictionary type.

Reply:
sorry @nand4011 this is my bad, I'm sure the way we talked about argument names yesterday was misleading and I left this part out.

Everything we talked about yesterday I think still holds for argument names, but for all of our Response.Hit objects, we have adopted the convention of always using the `.value` prefix for all data accessors.  Across all of the data types.  The intent is that any time a user is trying to get data out of a Response.Hit they will know that they can just type `.val` and then the autocomplete will show them all of the choices.

Reply:
Thanks for clarifying.